### PR TITLE
fix: pass project list of global module to local modules if not specified

### PR DESCRIPTION
**Pull request by maxpyf**
## Summary

Pass the project_filter_list from the global module to the local, regional modules if not otherwise specified. The documentation does not say to do this, so this is confusing for customers.

## How did you test this change?

- [x] Ran the internal CI tests. 
- [x] Tested on live multi-region org internally.

## Issue

N/A. Reported internally.

### CODE REVIEW

File: examples/custom-vpc-network/main.tf [hunk 1]
```diff
@@ -82,8 +82,6 @@ module "lacework_gcp_agentless_scanning_project_multi_region_usc1" {
     google = google.usc1
   }
 
-  project_filter_list = local.project_filter_list
```
Comment (theopolis) [inline examples/custom-vpc-network/main.tf:85]: Is this removed to test that the module works without supplying the filter list?

**Review by theopolis — APPROVED** [review]

File: examples/custom-vpc-network/main.tf (hunk 1 above)
Comment (maxpyf) [reply examples/custom-vpc-network/main.tf:85]: yes.

**Review by ammarekbote — APPROVED** [review]
Do we need to modify the example here https://github.com/lacework/terraform-gcp-agentless-scanning/blob/main/examples/project-level-multi-region/main.tf since there is no need to specify `filter_list` for each region now?

**Comment by maxpyf** [comment]
> Do we need to modify the example here https://github.com/lacework/terraform-gcp-agentless-scanning/blob/main/examples/project-level-multi-region/main.tf since there is no need to specify `filter_list` for each region now?

Good point, let me change that.

**Comment by maxpyf** [comment]
Closing this PR since merging is blocked by absence of signature - will retry.

**Pull request closed without merging**